examples/tests/valid/struct_from_json.w

```diff
@@ -236,4 +236,16 @@ test "lifting a student" {
   assert(studentInflight1.dob.month == 10);
   assert(studentInflight1.dob.day == 10);
   assert(studentInflight1.dob.year == 2005);
-}
\ No newline at end of file
+}
+
+// bring structs from other files
+bring "./subdir/structs.w" as externalStructs;
```

Comment:
Does this also work for structs imported via JSII? Maybe add a test..,

Reply:
Based on this https://github.com/winglang/wing/issues/3943 I take it does not and is being left for later

Reply:
Yupp thats exactly right @MarkMcCulloh, Im brainstorming the JSII imported structs now. The issue here is that we arent jsifying those so we never create a jsonschema for them to use for validation. I have a few ideas that are probably not gonna work but may 😁